src/Screens/Bookmark/BookmarkScreen.js

```diff
@@ -1,14 +1,66 @@
-import { View, Text } from "react-native";
+/* 
+  Bookmark Screen displays the bookmarks selected by client.
+  Source from Redux store; 
+*/
+
 import React from "react";
+import { View, Text, FlatList, Pressable } from "react-native";
+import styles from "../../Shared/styles";
+import { useSelector } from "react-redux";
+import Icon from "../../Shared/Components/Icon.js";
 import Breadcrumb from "../../Shared/Components/header/Breadcrumb";
+import { BookmarkCard } from "./BookmarkCard";
 
 const BookmarkScreen = () => {
-  return (
-    <View>
-      <Breadcrumb />
-      <Text>BookmarkScreen</Text>
-    </View>
-  );
+  const bookmarks = useSelector(state => state.bookmarks); //get bookmarks array
+
+  const renderBookmarkItem = ({ item }) => {
+    return <BookmarkCard item={item} />;
+  };
+
+  /*Output Section*/
+
+  if (bookmarks.bookmarkArray.length === 0) {
+    return (
+      <View>
+        <Breadcrumb />
+        <View style={[styles.bookmarkScreenFormatting, styles.centerOnScreen]}>
+          <Icon
+            iconStyle="Ionicons"
+            iconName="bookmarks-sharp"
+            style={styles.bookmarkIconLarge}
+          />
+          <Text style={styles.bookmarkNoneTitleRender}>No Bookmarks Yet</Text>
+          <View style={styles.bookmarkNoneLabelRender}>
+            <Text
+              style={[styles.bookmarkTextRender, styles.bookmarkNoneBoxRender]}
+            >
+              Keep track of your most popular articles and documents by clicking
+              the
+              <Icon
+                iconStyle="FontAwesome"
+                iconName="bookmark"
+                style={styles.bookmarkIconTiny}
+              />
+              icon.
+            </Text>
+          </View>
+        </View>
+      </View>
+    );
+  } else {
+    return (
+      <View>
+        <Breadcrumb />
+        <View style={styles.bookmarkScreenFormatting}>
+          <FlatList
+            data={bookmarks.bookmarkArray}
+            renderItem={renderBookmarkItem}
```

Comment:
Far as I know, renderItem can take components, so you don't need the renderBookmarkItem function to return your BookmarkCard.
```suggestion
            renderItem={(item) => <BookmarkCard item={item}/>}
```
See if something like this works.

Reply:
If I do it this way, I lose the Pressable event capability which breaks the navigation.

Reply:
Discussed verbally.  Keeping it the way I wrote it.